sacrebleu/metrics/base.py

```diff
@@ -325,12 +325,7 @@ def _cache_references(self, references: Sequence[Sequence[str]]) -> List[Any]:
         num_refs = set()
 
         for refs in zip(*references):
-            # remove undefined / empty references
-            # i.e. we have fewer references for this particular sentence
-            lines = [x for x in refs if x is not None and x != ""]
-
-            if len(lines) == 0:
-                raise RuntimeError("Empty or `None` reference sentence found.")
+            lines = [x for x in refs]
```

Comment:
You should keep the `None` check here to not break the variable number of references feature as discussed in both the related PR and the issue.

Reply:
Done, thanks for the catch!